packages/block/test/eip4844block.spec.ts

```diff
@@ -17,249 +17,253 @@ import { Block } from '../src/index.js'
-            },
-            {
-              common: earlyCommon,
-            }
-          )
-        },
-        'blob gas used can only be provided with EIP4844 activated',
-        undefined,
-        'should throw when setting blobGasUsed with EIP4844 not being activated'
-      )
+    assert.throws(
+      () => {
+        BlockHeader.fromHeaderData(
+          {
+            excessBlobGas: 1n,
+          },
+          {
+            common: earlyCommon,
+          }
+        )
+      },
+      'excess blob gas can only be provided with EIP4844 activated',
+      undefined,
+      'should throw when setting excessBlobGas with EIP4844 not being activated'
+    )
 
-      const excessBlobGas = BlockHeader.fromHeaderData(
-        {},
-        { common, skipConsensusFormatValidation: true }
-      ).excessBlobGas
-      assert.equal(
-        excessBlobGas,
-        0n,
-        'instantiates block with reasonable default excess blob gas value when not provided'
-      )
-      assert.doesNotThrow(() => {
+    assert.throws(
+      () => {
         BlockHeader.fromHeaderData(
           {
-            excessBlobGas: 0n,
+            blobGasUsed: 1n,
           },
           {
-            common,
-            skipConsensusFormatValidation: true,
+            common: earlyCommon,
           }
         )
-      }, 'correctly instantiates an EIP4844 block header')
+      },
+      'blob gas used can only be provided with EIP4844 activated',
+      undefined,
+      'should throw when setting blobGasUsed with EIP4844 not being activated'
+    )
 
-      const block = Block.fromBlockData(
+    const excessBlobGas = BlockHeader.fromHeaderData(
+      {},
+      { common, skipConsensusFormatValidation: true }
+    ).excessBlobGas
+    assert.equal(
+      excessBlobGas,
+      0n,
+      'instantiates block with reasonable default excess blob gas value when not provided'
+    )
+    assert.doesNotThrow(() => {
+      BlockHeader.fromHeaderData(
         {
-          header: BlockHeader.fromHeaderData({}, { common, skipConsensusFormatValidation: true }),
+          excessBlobGas: 0n,
         },
-        { common, skipConsensusFormatValidation: true }
-      )
-      assert.equal(
-        block.toJSON().header?.excessBlobGas,
-        '0x0',
-        'JSON output includes excessBlobGas'
+        {
+          common,
+          skipConsensusFormatValidation: true,
+        }
       )
-    }
+    }, 'correctly instantiates an EIP4844 block header')
+
+    const block = Block.fromBlockData(
+      {
+        header: BlockHeader.fromHeaderData({}, { common, skipConsensusFormatValidation: true }),
+      },
+      { common, skipConsensusFormatValidation: true }
+    )
+    assert.equal(block.toJSON().header?.excessBlobGas, '0x0', 'JSON output includes excessBlobGas')
   })
 })
 
 describe('blob gas tests', () => {
+  let common: Common
+  let blobGasPerBlob: bigint
+  beforeAll(async () => {
+    const kzg = await initKzg()
+    initKZG(kzg, '')
```

Comment:
do we name slighly different signature than `initKzg` (may be loadKzg?) seems like very similar to `initKZG`

Reply:
Yep, as discussed on our call, will rename this to `createKZG` in `kzg-wasm`